smartapps/samsung/sleepsense-in-bed.src/sleepsense-in-bed.groovy

```diff
@@ -0,0 +1,152 @@
+/**
+ *  SLEEPsense-in bed
+ *
+ *  Copyright 2015 Samsung Mobileapp
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+definition(
+    name: "SLEEPsense-in bed",
+    namespace: "samsung",
+    author: "Samsung Mobileapp",
+    description: "Control lights when you’re in bed.",
+    category: "My Apps",
+    iconUrl: "https://s3.amazonaws.com/smartapp-icons/Convenience/Cat-Convenience.png",
+    iconX2Url: "https://s3.amazonaws.com/smartapp-icons/Convenience/Cat-Convenience@2x.png",
+    iconX3Url: "https://s3.amazonaws.com/smartapp-icons/Convenience/Cat-Convenience@2x.png")
+
+
+preferences {
+	page(name:"Mainpage", install:true, uninstall:true) {
+        section("Control lights when you're in bed.") {
```

Comment:
Section title is inaccurate. This is the section where you select the select sensor.